lighthouse/blueprints/plates.py

```diff
@@ -26,10 +26,14 @@ def create_plate_from_barcode() -> Tuple[Dict[str, Any], int]:
         samples = get_positive_samples(barcode)
 
         if not samples:
-            return {"errors": ["No samples for this barcode"]}, HTTPStatus.BAD_REQUEST
+            return {"errors": ["No samples for this barcode: " + barcode]}, HTTPStatus.BAD_REQUEST
```

Comment:
Maybe it would be worth adding a test for this? I know there wasn't one for this before but it seems to have started failing code coverage.

Reply:
Yep I will have a look